inc/class_captcha.php

```diff
@@ -33,6 +33,13 @@
  *
  */
 
+define('DEFAULT_CAPTCHA', 1);
```

Comment:
How about `MYBB_DEFAULT` (when combined with the suggestion to make them contained within the class)?